base/exports.jl

```diff
@@ -1149,6 +1149,7 @@ export
     ntoh,
     open,
     pipe,
+    Pipe,
```

Comment:
Maybe the `pipe` function should be renamed instead? Calling this thing a "pipe" is pretty standard UNIX terminology.